TimeTrackerUI/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I'm sure you didn't want to change this file

Reply:
Added because of File.Create command not by me